data/models10.sample.go

```diff
@@ -0,0 +1,23 @@
+// Package models is a test
+package models
+
```

Comment:
Add import fmt

Reply:
That is an old commit and I have not changed it.